halo2-ecc/src/bn254/tests/bls_signature.rs

```diff
@@ -68,12 +65,11 @@ fn bls_signature_test<F: PrimeField>(
             .final_exponentiation();
 
     // Compare the 2 results
-    let fp12_chip = Fp12Chip::new(&fp_chip_1);
-    assert_eq!(
-        format!("Gt({:?})", fp12_chip.get_assigned_value(&result.into())),
-        format!("{actual_result:?}"),
-        "Signatures do not match!"
-    );
+    if *(result.value()) == 1.into() {
```

Comment:
I think this is fine; if you wanted to be fancy you could do `assert_eq!(*result.value(), Fr::from(actual_result == Gt::identity()))` but it's the same effect

Reply:
Thank you!